dict/dict0dict.cc

```diff
@@ -577,10 +566,12 @@ void dict_table_add_to_cache(dict_table_t *table, mem_heap_t *heap) {
   }
 
   /* Add table to hash table of tables */
-  HASH_INSERT(dict_table_t, name_hash, dict_sys->table_hash, fold, table);
+  auto table_name_hash_insertion = dict_sys->table_hash->emplace(table->name, table);
```

Comment:
The scope of the variable is 2 lines, doesn't make sense to use such a long name :-). It will suffice and scope it with {}.